donate.html

```diff
@@ -18,44 +17,35 @@
   #action-lead {
     max-width: 800px;
   }
-
-  #donation-other, #ak-recurring-type, #in-honor-of {
+  #donation-other, #ak-recurring-type, #in-honor-of, #site-footer, #admin-bar, #action-toc {

```

Comment:
just a note (mostly to myself) that we need to update the `if page.custom_fields.after_action_next_step` to check for a new custom page field that we create to enable donate embeds